spec/predicates/release.md

```diff
@@ -0,0 +1,102 @@
+# Predicate type: Release
+
+Type URI: https://in-toto.io/attestation/release
+
+Version 0.1
+
+## Purpose
+
+To authoritatively link from a specific release name and version string in a registry, to the artifact names and hashes that make up that release.
+
+## Use Cases
```

Comment:
Could you go into slightly more detail about what this is and is not intended to protect against? IIUC:

- Protects against a mirror substituting a malicious artifact (i.e. authenticity / integrity).
- Does not protect against a mirror pretending that a release, or an artifact within a release, does not exist (i.e. availability).

Not sure if there's more nuance to it than that.

Reply:
I added language around protecting integrity and not protecting availability.